docs/docs/destinations/sql/README.md

```diff
@@ -357,7 +357,7 @@ If you want to test potential table names against a [regular expression](https:/
 
```

Comment:
(didn't update this one to reference component actions since we're deprecating soon)